pyvista/plotting/plotting.py

```diff
@@ -5406,6 +5516,10 @@ def show(
 
         self.render()
 
+        # initial double render needed for certain passes when offscreen
+        if self.off_screen and 'vtkDepthOfFieldPass' in self.renderer._render_passes._passes:
+            self.render()
```

Comment:
Codecov said that this branch was uncovered, not sure why. Does that make sense? Or is it just that we have no `plotter.show()` with DOF in our test suite? `test_many_multi_pass()` should cover that case, so I'm not sure why this wasn't entered.

By the way, I reversed the order of the two conditions here, because `self.off_screen` should be `False` most of the time in user code, short-circuiting.

Reply:
Covered now.